packages/covector/__snapshots__/index.test.ts.snap

```diff
@@ -1514,6 +1514,721 @@ Object {
 }
 `;
 
+exports[`integration test for preview command runs version and publish for js and rust 1`] = `
+Object {
+  "consoleInfo": Array [],
+  "consoleLog": Array [
+    Array [
+      "Checking if tauri-bundler@0.6.0 is already published with: echo 0.5.2",
```

Comment:
Do we have this enabled for rust? Seems like it these are not respecting the preview package version.

Reply:
It [should](https://github.com/jbolda/covector/blob/mk/run-preview/packages/apply/index.ts#L312-L317) bump for both rust and js. Tauri-bundler shouldn't have been bumped with a preview version because there's no change file for it in this particular fixture.